README.rst

```diff
@@ -1,7 +1,11 @@
 BrowserID Page Object Model
 ===========================
 
-Selenium compatible page object model for
+BIDPOM is no longer in active development, and support has been discontinued.
+
+Documentation may be found `here <https://github.com/mozilla/bidpom/wiki>`_.
+
+Selenium-compatible page-object model for
 `Mozilla Persona <https://login.persona.org/>`_ (previously known as BrowserID).
```

Comment:
I would put the notice beneath the project description with a heading, such as https://github.com/mozilla/moz-grid-config#this-repository-is-no-longer-actively-maintained. I think it also makes sense to mention the reason, which is due to the retirement of Persona.